src/components/EpisodeCard.tsx

```diff
@@ -0,0 +1,38 @@
+import IEpisode from "../interfaces/IEpisode";
+import cleanString from "../util/cleanString";
+import padNumber from "../util/padNumber";
+import "./episodeCard.css";
+
+interface EpisodeCardProps {
+    key: number;
+    episode: IEpisode;
+}
+
+export function EpisodeCard(props: EpisodeCardProps): JSX.Element {
+    const e = props.episode;
+    const paddedSeasonNumber = padNumber(e.season);
+    const paddedEpisodeNumber = padNumber(e.number);
+    let cleanedSummary = cleanString(e.summary, "<p>");
+    cleanedSummary = cleanString(cleanedSummary, "</p>");
+    cleanedSummary = cleanString(cleanedSummary, "<br>");
+    cleanedSummary = cleanString(cleanedSummary, "<br />");
+    cleanedSummary = cleanString(cleanedSummary, "<b>");
+    cleanedSummary = cleanString(cleanedSummary, "</b>");
+    cleanedSummary = cleanString(cleanedSummary, "</i>");
+    cleanedSummary = cleanString(cleanedSummary, "<i>");
```

Comment:
This is repetitive and only works if you are sure of that these are the only tags you'll encounter.

You could use regular expressions to help avoid this, or a for loop.

Reply:
We know! No time!